dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/SdkFeatureTest.kt

```diff
@@ -280,7 +280,7 @@ internal abstract class SdkFeatureTest<T : Any, C : Configuration.Feature, F : S
     }
 
     @Test
-    fun `𝕄 migrate batch files 𝕎 initialize()`(
+    open fun `𝕄 migrate batch files 𝕎 initialize()`(
```

Comment:
Instead of making this method open, which seems weird for a test method, can you make an `abstract fun doesfeatureNeedMigration(): Boolean` and then in this test exit if this returns false ?